ksqldb-functional-tests/src/test/resources/query-validation-tests/avro.json

```diff
@@ -592,11 +592,11 @@
           ]}
         }
       ],
-      "inputs": [{"topic": "input", "value": {"expected": 1, "c2": null, "c3": null}}],
-      "outputs": [{"topic": "OUTPUT", "value": {"EXPECTED": 1}}],
+      "inputs": [{"topic": "input", "value": {"expected": 1, "c1": null, "c2": null, "c3": null}}],
```

Comment:
Bytes don't work in Avro/Protobuf tests yet because there is no real way to represent a `ByteBuffer` in JSON. Nulls are fine though because they're just represented by `null`